AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -622,9 +622,8 @@ extension MeetingModel: ActiveSpeakerObserver {
     }
 
     func activeSpeakerDidDetect(attendeeInfo: [AttendeeInfo]) {
-        videoModel.updateRemoteVideoStatesBasedOnActiveSpeakers(activeSpeakers: attendeeInfo)
         if activeMode == .video {
-            videoModel.videoUpdatedHandler?()
```

Comment:
can't you put alreadySorted logic in the `videoUpdatedHandler`?

Reply:
Not really, because videoUpdatedHandler is called from other places as well.